pgModel/models.go

```diff
@@ -0,0 +1,64 @@
+package pgModel
+
+type JwtToken struct {
+    Token string `json:"token"`
+}
+
+type Exception struct {
+    Message string `json:"message"`
+}
+
+type User struct {
+	Id     int64 `sql:",pk,unique"`
+	Username   string
+	Email string `sql:",unique,notnull"`
+	Password string
+	PhoneNumber string
+	Messages []*Message
+	Connections []*Connection
+	Comments []*Comment
+	Collections []*Collection
+}
+
+type Connection struct {
+	Id int64
+	FirstUserId int64
+	SecondUserId int64
+}
+
+type Message struct {
+	Id int64
+	UserId int64
+	ReceiverId int64
+	Content string
+}
+
+type Resource struct {
+	Id int64
+	CollectionId int64
+	Link string
+	Type string
+	Views int64
+	Recommendations int64
+	Comments []*Comment
+}
+
+type Comment struct {
+	Id int64
+	UserId int64
+	ResourceId int64
+	Message string
```

Comment:
@mikey2020 can we make this `Text` in place of `Message`?

Reply:
Also, the `VivaCount` was omitted. I'm not sure we've totally agreed on this yet.
Perhaps we should just go with `Likes` or `LikeCount` then make a decision later as to what appears on the user interface.